lighthouse-core/audits/dobetterweb/notification-on-start.js

```diff
@@ -22,7 +22,7 @@ const UIStrings = {
   /** Description of a Lighthouse audit that tells the user why they should not ask for notification permission on load. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Users are mistrustful of or confused by sites that request to send ' +
       'notifications without context. Consider tying the request to user gestures ' +
-      'instead. [Learn more](https://web.dev/notification-on-start/).',
+      'instead. [Learn more about notification permission](https://web.dev/notification-on-start/).',
```

Comment:
```suggestion
      'instead. [Learn more about responsibility getting permission for notifications](https://web.dev/notification-on-start/).',
```

Reply:
Seems like a typo, did you mean `Learn more about responsibly getting permission for notifications`?

Reply:
hah yes.